platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationPropertyReferencedEntryImpl.java

```diff
@@ -77,6 +91,26 @@ public boolean restore(boolean required) {
   }
 
   @Override
+  public boolean restore(boolean required, PathMatcher filter) {
+    Validate.notNull(filter, "invalid null path filter");
+    if (restored == null) {
+      super.restored = false; // until proven otherwise
```

Comment:
I don't think you need isDebugEnabled() with slf4j. https://www.slf4j.org/faq.html#logging_performance

Reply:
see above replies

Reply:
Also...their faq is painfully naive. This for example, would be a very expensive noop log message if debug not enabled:
```java
LOGGER.debug("This is trivial: {}", expensiveOperation());
...
public String expensiveOperation() {
    try {
        Thread.sleep(10000000000000L);
    catch (Exception ignore) {}
    return "slf4j is wrong";
}
```